src/main/kotlin/com/makarytskyi/rentcar/model/Repairing.kt

```diff
@@ -0,0 +1,25 @@
+package com.makarytskyi.rentcar.model
+
+import com.makarytskyi.rentcar.dto.repairing.RepairingResponse
+import java.util.Date
+
+data class Repairing(
+    val id: String? = null,
+    val carId: String?,
+    val date: Date?,
+    val price: Int?,
+    val status: RepairingStatus?,
+) {
+
+    enum class RepairingStatus {
+        PENDING, IN_PROGRESS, COMPLETED;
+    }
+
+    fun toResponse(): RepairingResponse = RepairingResponse(
+        id ?: "none",
```

Comment:
here it is reasonable to use "!!" as I can't imagine situation when id would be null